<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget"/>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenMesh"/>
    <addaction name="actionCutting"/>
    <addaction name="actionShowL"/>
    <addaction name="actionShowR"/>
    <addaction name="actionShowToothAndGingiva"/>
    <addaction name="actionOpenMeshFolder"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpenMesh">
   <property name="text">
    <string>Open Mesh File</string>
   </property>
  </action>
  <action name="actionCutting">
   <property name="text">
    <string>Cutting</string>
   </property>
  </action>
  <action name="actionShowL">
   <property name="text">
    <string>ShowTooth</string>
   </property>
  </action>
  <action name="actionShowR">
   <property name="text">
    <string>ShowGingiva</string>
   </property>
  </action>
  <action name="actionShowToothAndGingiva">
   <property name="text">
    <string>ShowToothAndGingiva</string>
   </property>
  </action>
  <action name="actionOpenMeshFolder">
   <property name="text">
    <string>Open Mesh Folder</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
